File: LLD Solutions/Bowling-Alley-Machine-Coding-Flipkart-Interview/Controller/Player.cs
using System;

namespace Bowling_Alley
{
    internal class Player
    {
        private string name;
        private bool isWinner;
        private int round;
        private PlayerScoreCard playerScoreCard;
        public Player(string name)
        {
            this.name = name;
            playerScoreCard = new PlayerScoreCard();
        }

        public bool IsWinner()
        {
            return isWinner;
        }

        public string getName()
        {
            return name;
        }

        public void setRound(int round)
        {
            this.round = round;
        }
        public int getRound()
        {
            return this.round;
        }

    }
}

File: Low_level_Problem_set_2/bowlingAlley/model/Player.java
package bowlingAlley.model;

/**
 * @author priyamvora
 * @created 23/05/2021
 */
public class Player {
    private final String playerId;
    private final String name;

    public Player(String playerId, String name) {
        this.playerId = playerId;
        this.name = name;
    }

    @Override
    public String toString() {
        return "Player{" +
                "playerId='" + playerId + '\'' +
                ", name='" + name + '\'' +
                '}';
    }

    public String getPlayerId() {
        return playerId;
    }

    public String getName() {
        return name;
    }
}
